extra/utils/Validation.js

```diff
@@ -43,6 +30,10 @@ function validateUsers(users){
     console.log("Valid is :"+a);
     console.log("invalid is :" +b);
     console.log("Valid: "+countValid+" , Invalid: "+countInvalid);
-};
+}
     
-validateUsers(users);
\ No newline at end of file
+
+//validateUsers(users);
+
+validateUsers(users);
```

Comment:
no need to execute function here

Reply:
I checked this files and I found it. then I am removed it.